### PR TITLE
Fix template naming in check_template() and remove pkg imports

### PR DESCRIPTION
**Pull request by trianglegirl**

### CODE REVIEW

File: NAMESPACE [hunk 1]
```diff
@@ -1,5 +1,6 @@
 # Generated by roxygen2: do not edit by hand
 
+export("%>%")
```
Comment (csgillespie) [inline NAMESPACE:3]: Do we really want to export %>% ? It could potentially hide an issue when building notes if we forget to do library(dplyr), et.c

Comment (trianglegirl) [reply NAMESPACE:3]: Good point. Can remove - however this means importing all of dplyr again (and increasing allowed imports to 1) Is that our preference?

Comment (csgillespie) [reply NAMESPACE:3]: Why do we need to import all of dplyr not just %>% from magrritr?

Comment (trianglegirl) [reply NAMESPACE]: Removed `@export` and `@usage` which was causing an error.

**Review by csgillespie — APPROVED** [review]

**Pull request merged**